hopsworks-common/src/main/java/io/hops/hopsworks/common/user/AuthController.java

```diff
@@ -463,6 +604,11 @@ public void registerFalseLogin(Users user, HttpServletRequest req) {
       userFacade.update(user);
     }
   }
+  
+  private boolean isUserAgent(Users user) {
```

Comment:
We already have `isUserInRole` method. We should reuse that one.

Reply:
just a private method, I think it is more readable this way